tests/devices/test_default_mixed_jax.py

```diff
@@ -667,7 +667,7 @@ def test_batching(self, jacobian_fn, decorator, tol):
             # TODO: https://github.com/PennyLaneAI/pennylane/issues/2762
             pytest.xfail("Parameter broadcasting currently not supported for JAX jit")
```

Comment:
unrelated, but this bug is closed as fixed. I removed this xfail locally on master and it still fails - I will try to check it out later, just commenting to get visibility

Reply:
Good catch let's take a look next week during feature freeze!

Reply:
:grimacing: